app/src/main/java/org/oppia/app/topic/revisioncard/RevisionCardActivityPresenter.kt

```diff
@@ -84,7 +96,7 @@ class RevisionCardActivityPresenter @Inject constructor(
     subtopicLiveData.observe(
       activity,
       Observer<String> {
-        activity.supportActionBar?.title = it
+        revisionCardToolbarTitle.text = it
```

Comment:
You can hard-code string here something like:
`Long toolbar title for testing purposes`